### ISSUE TITLE
update data import/export to use Docker API functionality rather than current hack

### ISSUE THREAD

**Issue by compleatang** [eris-ltd/eris-cli#131]
use new docker API support for this feature (with backwards compatibility)...

depends upon: https://github.com/fsouza/go-dockerclient/issues/352 being successful closed.

**Comment by compleatang** [comment]
apparently PR started :+1:

**Comment by compleatang** [comment]
this is unblocked. PR merged into the lib.

**Comment by compleatang** [comment]
Zach, bumping this across to your bucket.

**Comment by zramsay** [comment]
#315
